rest_framework/views.py

```diff
@@ -482,9 +482,9 @@ def raise_uncaught_exception(self, exc):
     # Note: Views are made CSRF exempt from within `as_view` as to prevent
     # accidental removal of this exemption in cases where `dispatch` needs to
     # be overridden.
-    def dispatch(self, request, *args, **kwargs):
+    def sync_dispatch(self, request, *args, **kwargs):
```

Comment:
Thoughts on subclassing `APIView` into `AsyncAPIView` ?

Reply:
That was specifically discouraged here https://github.com/encode/django-rest-framework/issues/8496#issuecomment-1226896860